scholarx/archive/functions.js

```diff
@@ -2,6 +2,8 @@ $(function () {
     loadNavAndFooter('/assets/content/static');  //relative path to content directory
 });
 
+let mentors = []
+let mentees = []
```

Comment:
Can you add a new line after this, so we can see the difference between the code lines well @janithlahirukariyawasam 
```suggestion
let mentees = []

```